File: DailyChallenge/Python/20260105_TirePressure.py
def tire_status(pressures_psi, range_bar):
    bar = 14.5038

    statusLow = "Low"
    statusGood = "Good"
    statusHigh = "High"

    tireStatus = []

    for index, tirePressure in enumerate(pressures_psi):
        minPressure, maxPressure = range_bar

        minPressure *= bar
        maxPressure *= bar

        if tirePressure < minPressure:
            tireStatus.append(statusLow)
        elif tirePressure > maxPressure:
            tireStatus.append(statusHigh)
        else:
            tireStatus.append(statusGood)

    return tireStatus

File: DailyChallenge/JavaScript/20260105_TirePressure.js
function tireStatus(pressuresPSI, rangeBar) {
  const bar = 14.5038;

  const statusLow = "Low";
  const statusGood = "Good";
  const statusHigh = "High";

  let tireStatus = [];

  for(let i = 0; i < pressuresPSI.length; i++) {
    let tirePressure = pressuresPSI[i];

    let minPressure = rangeBar[0] * bar;
    let maxPressure = rangeBar[1] * bar;

    if(tirePressure < minPressure) {
      tireStatus[i] = statusLow;
    }
    else if(tirePressure > maxPressure) {
      tireStatus[i] = statusHigh;
    }
    else {
      tireStatus[i] = statusGood;
    }
  }

  return tireStatus;
}